src/components/sidebar/Sidebar.tsx

```diff
@@ -39,8 +40,27 @@ class Sidebar extends React.Component<ISidebarProps, ILinelistState> {
             const homeLink = routeBuilder.to(subSites.home).clear().toLink();
             navigator.goTo(homeLink);
         };
+
+        const getContentComponent = () => {
+            if (this.props.sidebarStore!.showNodeWindow) {
+                return <NodeWindow />;
+            }
+            if (this.props.sidebarStore!.showLinkWindow) {
+                return <LinkWindow />;
+            }
+            return (
+                <Switch>
+                    <Route path={subSites.routes} component={RoutesView} />
+                    <Route path={subSites.home} component={HomeView} />
+                </Switch>
+            );
+        };
+
         return (
-            <div className={s.sidebarView}>
+            <div
+                className={s.sidebarView}
+                style={{ width: this.props.sidebarStore!.getSideBarWidth }}
```

Comment:
Should we just agree on a set of different sizes to avoid settings styles like this?